roles/identity-management/manage-idm-identities/tasks/create_users.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+- name: "Process users for IPA/IdM"
+  block:
+
+    - name: Create IPA/IdM user
+      ipa_user:
+        ipa_host: "{{ ipa_host | default(ansible_host)}}"
+        ipa_user: "{{ ipa_admin_user }}"
+        ipa_pass: "{{ ipa_admin_password }}"
+        validate_certs: "{{ ipa_validate_certs | default(omit) }}"
+        givenname: "{{ user_data.first_name | trim }}"
+        sn: "{{ user_data.last_name | trim }}"
+        name: "{{ user_data.user_name | trim }}"
+        mail: "{{ user_data.email | default(omit) }}"
+        state: "{{ user_data.state | default(omit) }}"
+        # The addition of expiration date is a request that will be submitted upstream
+        #krbprincipalexpiration: "{{ user.expiration_date | default(omit) }}"
```

Comment:
Can we just clean up expiratoin_date to match everything else while we're here? Otherwise we'll need to come back and address this later. I think this should just be `user_data.expiration_date` unless we foresee using a more global expiration date. Might be useful to allow setting one at the user level and then defaulting to a global one if the user isn't present (or to empty if neither of them are set).

Reply:
Fixed this one